target/linux/lantiq/files/arch/mips/boot/dts/lantiq/ar9.dtsi

```diff
@@ -44,8 +44,8 @@
 			#interrupt-cells = <1>;
 			interrupt-controller;
 			compatible = "lantiq,icu";
-			/* TODO: AR9 should have ICU1 (like VR9) too */
-			reg = <0x80200 0xc8>;
+			reg = <0x80200 0xc8	/* icu0 */
+				0x80300 0xc8>;	/* icu1 */
```

Comment:
This looks good to me, if there are two VPEs there should aslo be two ICU blocks.

Reply:
Is there any ar9 device which uses the voice FW? This firmware should be loaded into the 2. VPE.

Reply:
The FXS ports on all ar9 devices are unsupported and there doesn't seem to be any development towards a driver and firmware support, so I thought it would be more useful to use the second VPE for OpenWRT, too.